<launch>

  <node pkg="zeta2_bringup" exec="zeta_mc_node"  name="zeta_mc" output="screen">
    <param name="port_name" value="/dev/ttyUSB1"/>
    <param name="baudrate" value="115200"/>
  </node>
  
</launch>
